packages/updating-element/src/updating-element.ts

```diff
@@ -637,6 +689,14 @@ export abstract class UpdatingElement extends HTMLElement {
         (options.converter as ComplexAttributeConverter)?.toAttribute ??
         defaultConverter.toAttribute;
       const attrValue = toAttribute!(value, options.type);
+      if (DEV_MODE && attrValue === undefined) {
+        console.warn(
+          `The attribute value for the ` +
+            `${name as string} property is undefined. The attribute will be ` +
```

Comment:
Hmm, this doesn't indicate an error state, so this is more of a migration helper warning. Seems like you should be able to write code to the new rule (set properties to `undefined` and have them removed) without seeing a warning every time. The other warnings are always errors.  Maybe we need another debug warning level which is like "help me migrate".

Reply:
Done